### ISSUE TITLE
Prevent user from deleting custom invoice

### ISSUE THREAD

**Issue by tswfi** [TampereHacklab/mulysa#276]
Users can do strange things like create invoice pay it and delete it before mulysa knows about the payment.

Prevent users from deleting custom invoices. 

Bonus feature: add a "invalid" bit for invoices so that they can be shown in a different view  (invalid invoices) or something.

**Comment by olmari** [comment]
So how is user then supposed to ever get rid of such? I feel the recent "let's make custom invoices not be there that prominent" should be either enough or at least nice start. With this one is only asking more management troubles for later on I feel :)

**Comment by tswfi** [comment]
thats where the "invalid" bit comes in.

if the invoice is completely deleted figuring out the incoming transaction becomes impossible as there is no data.. (or the transaction has to be just marked as "donation to hacklab")

**Comment by olmari** [comment]
Indeed... Totally deleting from system is propably not good idea, at least custom invoice alone etc... Things to be thought still that how does the view show "invalid" or not and possibly where / to whom (users, admins).. For long run maybe some "past invoices" or "past subscriptions" would be most proper place to show such "past events" that has any meaning despite event itself being "removed" from normal view.
